logic/strategy/Cargo.toml

```diff
@@ -18,15 +18,18 @@ async-std = { workspace = true, features = ["unstable"]}
 futures = { workspace = true }
 rand = "0.8.5"
 serde = { workspace = true, features = ["derive"] }
-serde_with = "3.3.0"
+serde_with = { workspace = true }
 lazy_static = { workspace = true }
+semver = "1.0"
 strum = { workspace = true }
 thiserror = { workspace = true }
 validator = { workspace = true }
 
+hopr-crypto-random = { workspace = true }
 hopr-crypto-types = { workspace = true }
 core-network = { workspace = true }
 core-protocol = { workspace = true }
+core-transport = { workspace = true }
```

Comment:
This should be abstracted by some traits.

Reply:
Once refactoring core-transport and core-network, we should also improve the peer metadata.
This dependency is brought in just for a single constant.

Reply:
I know, it will be better.